.github/workflows/publish.js.yml

```diff
@@ -23,13 +23,22 @@ jobs:
     - run: npm run test
       name: Run test
     - run: |
-        npm prune --production
-        rm -rf node_modules/appium
-      name: Remove dev dependencies and appium peer dependencies
-    - run: npm shrinkwrap # this command removes package-lock.json...
-      name: Create shrinkwrap
-    - run: npm ci --only=dev
-      name: Install dev dependencies for the release
+        # Remove dev and peer dependencies from node_modules
+        npm prune --omit=dev --omit=peer
+
+        # Generate shrinkwrap from the content of node_modules
+        mv package-lock.json package-lock.json.bak
+        npm shrinkwrap
```

Comment:
mm, it seems like this command puts `"dev"` in the npm-shrknkwrap.json? Let me check a few further later

Reply:
maybe the behaviour has changed in npm itself as we use the latest LTS node

Reply:
Checked a few additionally. Actually npm included in over node 18 (so lockfileVersion v3 thing?) generated `dev` included one. Thus currently no good way to include only `dependencies` without `dev` attribute in the published package, while actually the lock file with --omit (in current our publish.yml way) did not include everything in the dev dependencies, thus not bad